### DISCUSSION TITLE
does bookmarklet work

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#946] [(B) Bug Discussion]
bookmarklet dose not work for me please help-do i need redirector-when i click on bookmarklet nothing happens but when i drag it onto scrreen and new window opens up saying error#blocked

**Comment by ghost** [comment]
allso for dev tools hacks to work do i need redirector?

**Comment by LeoBadeaux** [comment]
There is no official nor unofficial bookmarklet, a bookmarklet does not and never will work. Please install our extensions from the Chrome Web Store.
